frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlertColumn.tsx

```diff
@@ -0,0 +1,39 @@
+import { cellWidth, expandable, IFormatter, ITransform, sortable } from '@patternfly/react-table';
+import { TFunction } from 'i18next';
+
+type MonitoringAlertColumn = {
+  title: string;
+  cellFormatters?: IFormatter[];
+  transforms?: ITransform[];
+  fieldName?: string;
+  sortFunc?: string;
+};
+
+export const MonitoringAlertColumn = (t: TFunction): MonitoringAlertColumn[] => [
```

Comment:
We use this in different components, I'm unsure if this is the easiest solution, but I can live with this until we get some performance issues or want to change this on all components at the same time.